canvas_sdk/data/base.py

```diff
@@ -0,0 +1,15 @@
+import json
+
+from pydantic import BaseModel, ConfigDict
+
+from canvas_sdk.utils.json import JSONEncoder
+
+
+class PluginModel(BaseModel):
```

Comment:
Let's just name this `Model` (everything in here is in reference to a plugin)

Reply:
Done.